scripts/core/get-superdesk-api-implementation.tsx

```diff
@@ -119,11 +129,33 @@ export function getSuperdeskApiImplementation(
                     });
                 },
             },
+            contentProfile: {
+                get: (id) => {
+                    // Adding simple caching since the function will be called multiple times per second.
+
+                    // TODO: implement synchronous API(and a cache) for accessing
```

Comment:
are you fixing that?

Reply:
Not in this PR. It affects this task, but the problem isn't unique to it.